### PR TITLE
icemon 3.1.0 (new formula) 

### PR DESCRIPTION
**Pull request by Sean-Der**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
Undefined symbols for architecture x86_64:
  "_lzo1x_1_compress", referenced from:
      MsgChannel::writecompressed(unsigned char const*, unsigned long, unsigned long&) in libicecc.a(libicecc_la-comm.o)
  "_lzo1x_decompress", referenced from:
      MsgChannel::readcompressed(unsigned char**, unsigned long&, unsigned long&) in libicecc.a(libicecc_la-comm.o)
```

File: Formula/icemon.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Icemon < Formula
+  desc "Icecream GUI Monitor"
+  homepage "https://github.com/icecc/icemon"
+  url "https://github.com/icecc/icemon/archive/v3.1.0.tar.gz"
+  sha256 "8500501d3f4968d52a1f4663491e26d861e006f843609351ec1172c983ad4464"
+  revision 1
```
Comment (fxcoudert) [inline Formula/icemon.rb]: No revision line

File: Formula/icemon.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  depends_on "cmake" => :build
+  depends_on "qt"
+  depends_on "icecream"
```
Comment (fxcoudert) [inline Formula/icemon.rb]: Alphabetical ordering among non-build deps

File: Formula/icemon.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  def install
+    args = std_cmake_args
+
+    system "cmake", ".", *args
+    system "make"
+    system "make", "install"
+  end
+end
```
Comment (fxcoudert) [inline Formula/icemon.rb]: This will need a functional test, in a `test do` block

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by Sean-Der** [comment]
@fxcoudert all green! thanks for the review

**Comment by Sean-Der** [comment]
Hey @fxcoudert @ilovezfs anything else I need to do, no rush though!

thanks

File: Formula/icemon.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  sha256 "8500501d3f4968d52a1f4663491e26d861e006f843609351ec1172c983ad4464"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "icecream"
+  depends_on "lzo"
+  depends_on "qt"
+
+  def install
+    args = std_cmake_args
```
Comment (DomT4) [inline Formula/icemon.rb]: Just inline this below since you're not adding extra arguments.

File: Formula/icemon.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    system "cmake", ".", *args
+    system "make"
```
Comment (fxcoudert) [inline Formula/icemon.rb]: Can this be combined into a single `make install` line?

**Comment by Sean-Der** [comment]
@BrewTestBot test this please

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @Sean-Der for the pull request and congrats on your first contribution to Homebrew!

**Pull request closed without merging**